.gitlab-ci.ornl.yml

```diff
@@ -0,0 +1,80 @@
+
+stages:
+  - build
+  - test
+  - clean
+
+Spack Build:
+  stage: build
+  tags:
+    - nobatch
+  needs: []
+  variables:
+#    CUSTOM_CI_BUILDS_DIR: "/gpfs/wolf/proj-shared/csc449/ci/${CI_PIPELINE_ID}"
+    WORK: "/gpfs/wolf/proj-shared/csc449/ci/"
+  script:
+    - pwd
+    - env | grep CI_
+    - export run_id=$CI_PIPELINE_ID
+    - export branch=$CI_COMMIT_BRANCH
+    - export url="https://sdk.testing.exaworks.org/result"
+    - export test="Spack"
+    - export location="ornl-$(echo $CI_RUNNER_TAGS | cut -f 1 -d ',')"
+    - export contact="andre@merzky.net"
+#    - export imnumber="LLNL-MI-834241"
+    - python3 ci/tests/test.py -s
+    - test -d $WORK/spack || git clone -c feature.manyFiles=true https://github.com/spack/spack.git $WORK/spack
```

Comment:
should be in sync with either (1) install and then removed or (2) installed once (in offline) and kept updated. Thus to have an up-to-date copy of Spack
https://github.com/ExaWorks/SDK/blob/04007917a86f77f6f51d0700a6d4507843a9a965/.gitlab/llnl-ci.yml#L8-L17

p.s. also would like to refer to have an option `-c core.sharedRepository=true` and to use `chmod -fR 02770 .` (see [LLNL CI readme](https://github.com/ExaWorks/SDK/blob/master/.gitlab/llnl-readme.md#environment-preparation-for-llnl-machines)) - thus anyone from our group (`exaworks`) can execute these commands. Before that we had permissions issue, since only one user was allowed to make changes

Reply:
Done